crates/pipeline-manager/src/config.rs

```diff
@@ -17,28 +17,23 @@ use std::{
 
 /// The default `platform_version` is formed using three compilation environment variables:
 /// - `CARGO_PKG_VERSION` set by Cargo
-/// - `FELDERA_ENTERPRISE_VERSION` set by the custom `build.rs` script,
-///   which is determined using the similarly named environment variable
 /// - `FELDERA_PLATFORM_VERSION_SUFFIX` set by the custom `build.rs` script,
 ///   which is determined using the similarly named environment variable
 ///
 /// ... and whether the `feldera-enterprise` feature is enabled.
 fn default_platform_version() -> String {
     let suffix = env!("FELDERA_PLATFORM_VERSION_SUFFIX").to_string();
+    let version = env!("CARGO_PKG_VERSION").to_string();
```

Comment:
This will be `pipeline_manager` version even if it is a dependency for another crate, as such this method doesn't work for downstream that use it as dependency.

Reply:
Yes, because the versions of OSS and enterprise are the same now so pipeline-manager can be the "ground truth"